dart-api-dl-sys/Cargo.toml

```diff
@@ -12,11 +12,11 @@ static_assertions = "1.1.0"
 
 [build-dependencies]
 bindgen = "0.59.2"
-cc = "1.0.72"
-semver = "1.0.4"
+cc = "1.0.73"
+semver = "1.0.5"
 
-# v0.1.20 is on edition 2021 which is a problem as it means rust won't parse the
+# v0.2.0 is on edition 2021 which is a problem as it means rust won't parse the
 # the Cargo.toml manifest. We don't use it but cargo can't know that and as such
 # downloads the sources and tries to parse the manifest.
 [target.'cfg(target_os = "hermit")'.build-dependencies]
-hermit-abi = ">=0.1.6, <0.1.20"
+hermit-abi = ">=0.1.6, <=0.1.19"
```

Comment:
This is more restrictive then needed. In case '0.1.19+1' is release we cannot pick it up.